automation_tools/satellite6/hammer.py

```diff
@@ -386,3 +386,21 @@ def attach_subscription_to_host_from_satellite(
         organization_id, product_name)
     return hammer('host subscription attach --subscription-id {0} '
                   '--host {1}'.format(subscription_id, hostname))
+
+
+def hammer_determine_cv_and_env_from_ak(ak_name, organization_id):
```

Comment:
Maybe renaming function to `hammer_determine_cv_and_env_from_ak`.

Also about the var name, most of times I add a suffix to make type explicity: `ak_params_dct`.

But once the code is small and type are described on docstring, I agree with @elyezer that changes are optional ;).